common/pb/session_commands.proto

```diff
@@ -105,6 +106,16 @@ message Command_JoinRoom {
     optional uint32 room_id = 1;
 }
 
+message Command_ForgotPassword{
+	extend SessionCommand {
+        optional Command_ForgotPassword ext = 1021;
+    }
+	required string user_name = 1;
+	required string password = 2;
```

Comment:
Is this for a desired new password? I'm not sure we should do that. We should probably have the server use a password reset process